### ISSUE TITLE
Can these files be removed?

### ISSUE THREAD

**Issue by jhamfler** [hftl-ims-research/wonder#32]
merge.bat
merge.sh
wonder_conversation.js
wonder_core.js
wonder_full.js

They are not needed/used. Classes and functions should be realoaded dynamically in the future and they provide everything in one file. Additionally they are giving too much not needed output when grep or something similar is used to search for occurencies of different functions or variables.

**Comment by sdruesedow** [comment]
yes, they can be removed. They where just attempts to reduce the amount of javascript files that the app-developer has to include. --> Obsolete
